packages/inter-protocol/src/econCommitteeCharter.js

```diff
@@ -38,12 +42,24 @@ const ParamChangesOfferArgsShape = M.splitRecord(
 );
 
 /**
- * @param {ZCF<{binaryVoteCounterInstallation:Installation}>} zcf
+ * A pattern for Zoe to check custom terms before `start()`ing the contract.
+ */
+export const customTermsShape = harden({
```

Comment:
This is unused.

Reply:
ditto from https://github.com/Agoric/agoric-sdk/pull/6699/